### PR TITLE
adjust author and version to current status.

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -24,11 +24,11 @@ use youki::commands::start;
 use youki::commands::state;
 use youki::rootless::should_use_rootless;
 
-/// High-level commandline option definition
-/// This takes global options as well as individual commands as specified in [OCI runtime-spec](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md)
-/// Also check [runc commandline documentation](https://github.com/opencontainers/runc/blob/master/man/runc.8.md) for more explanation
+// High-level commandline option definition
```
Comment (utam0k) [inline src/main.rs:27]: This is to prevent it from being output by the help command.

**Comment by nimrodshn** [comment]
LGTM 💯

File: src/main.rs (hunk 1 above)
Comment (YJDoc2) [reply src/main.rs:27]: lgtm :+1:

**Review by YJDoc2 — APPROVED** [review]
lgtm :+1:

**Pull request merged**